### ISSUE TITLE
round function not working

### ISSUE THREAD

**Issue by caternuson** [adafruit/circuitpython#410]
```python
Adafruit CircuitPython 2.1.0 on 2017-10-17; Adafruit CircuitPlayground Express with samd21g18
>>> round(3)
3
>>> round(3.0)
0
>>> round(3.8)
0
```

**Comment by jerryneedell** [comment]
odd - FYI - you do have access to several functions in the math module. ```import math
math.floor()
math.ceil()
math.trunc()

**Comment by jerryneedell** [comment]
round also fails the same in the master 3.0 build.

**Comment by dhalbert** [comment]
There is apparently a bug in https://github.com/adafruit/circuitpython/blob/master/lib/libm/nearbyintf.c in the builtin libm.

**Comment by dhalbert** [comment]
Fixed by #413.
